.github/workflows/call_sailbot_workspace_CI.yml

```diff
@@ -0,0 +1,25 @@
+name: Use sailbot workspace CI in other repos
+
+# This uses workflows in a different repo, refer to:
+# https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow
+
+# This runs sailbot workspace workflows and actions.
+# Runs all workflows within sailbot_workspace
+   
+# Run condition
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    # Every Friday at 1:00am UTC
+    - cron: '0 1 * * 5'
+  
+jobs:
+# Calls our workflows in sailbot_workspace 
+# UBCSailbot/sailbot_workspace/tree/main/.github/workflows
+  call-workflow-1-in-sailbot:
+        uses: UBCSailbot/sailbot_workspace/.github/workflows/tests.yml@user/colinli02/11-Publish-ROS-package-actions_FINAL
+        with:
+            repository: network_systems
```

Comment:
Generalize so that this can be used in any subteam repo: I think `${{ github.event.repository.name }}` should work?

Reply:
Or `${{ github.repository }}`, not sure which one is better

Reply:
Actually it should match https://github.com/UBCSailbot/sailbot_workspace/blob/d9429e7bbdde97c618e32c391a51f4ddf7d58ef4/.github/workflows/tests.yml#L27, which currently uses `${{ github.repository }}`

Reply:
Actually actually see https://github.com/UBCSailbot/sailbot_workspace/pull/100#discussion_r1118139809

Reply:
Done, used 
```
${{ github.event.repository.name }}
```

I made two separate commits, one with this and one without, can do a git revert in case ${{ github.event.repository.name }} doesn't work.